.travis.yml

```diff
@@ -20,6 +20,8 @@ script:
   - go get github.com/v3io/v3io-go-http
   - go get github.com/nuclio/logger
   - go get github.com/nuclio/nuclio-sdk-go
+  - cd $GOPATH/src/github.com/nuclio/nuclio-sdk-go
```

Comment:
I would rather
`cd X && do y && cd -`
to ensure shell pwd history is popped correctly and working dir isnt changed